docs/metadata.md

```diff
@@ -3,12 +3,12 @@ title:  |
   ![](assets/images/lightbits-cover-page.jpg){width=15cm}
 
 
-  Lightbits CSI Plugin v1.16.0 Deployment Guide
+  Lightbits CSI Plugin v1.17.0 Deployment Guide
 subtitle: |
 
   LightOS Version: v3.10.1
 
-  Kubernetes Versions: v1.17 - v1.24
+  Kubernetes Versions: v1.17 - v1.30
```

Comment:
@rahman-lb seems like support for version 30 was not merged to our master?

Reply:
@ronen-lb  deployment manifest for v25-v30,   https://github.com/LightBitsLabs/los-csi/pull/60

Reply:
@rahman-lb why not add as commits to this PR?

Reply:
@ronen-lb I wanted to separate the PR because 
   - If something goes wrong in support version from v1.25 -v1.30 , we don't need to change anything on PR  bumping version (except version v.130 mentioned) 
   - Any new CSI release uses previous version's bumping PR as references, and i wanted to keep the reference PR as much clean as possible.  
   -